Calibration/EcalAlCaRecoProducers/src/SelectedElectronFEDListProducer.cc

```diff
@@ -566,6 +569,35 @@ void SelectedElectronFEDListProducer<TEle,TCand>::pixelFedDump( std::vector<Pixe
  return ;
 }
 
+template< typename TEle, typename TCand>
+void SelectedElectronFEDListProducer<TEle,TCand>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<vector<edm::InputTag>>("electronTags",{edm::InputTag("hltEgammaGsfElectrons")});
+  desc.add<vector<edm::InputTag>>("recoEcalCandidateTags",{edm::InputTag("hltL1EG25Ele27WP85GsfTrackIsoFilter")});
+  desc.add<edm::FileInPath>("ESLookupTable",edm::FileInPath("EventFilter/ESDigiToRaw/data/ES_lookup_table.dat"));
```

Comment:
@diguida <https://github.com/diguida>, experts say that there is not such tag for ES at the moment. Also the ESDigiToRaw used that “hardcoded” lookup table


> Il giorno 09/mar/2015, alle ore 18:47, Salvatore Di Guida <notifications@github.com> ha scritto:
> 
> In Calibration/EcalAlCaRecoProducers/src/SelectedElectronFEDListProducer.cc <https://github.com/cms-sw/cmssw/pull/8112#discussion_r26058051>:
> 
> > @@ -566,6 +569,35 @@ void SelectedElectronFEDListProducer<TEle,TCand>::pixelFedDump( std::vector<Pixe
> >   return ;
> >  }
> >  
> > +template< typename TEle, typename TCand>
> > +void SelectedElectronFEDListProducer<TEle,TCand>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
> > +  edm::ParameterSetDescription desc;
> > +  desc.add<vector<edm::InputTag>>("electronTags",{edm::InputTag("hltEgammaGsfElectrons")});
> > +  desc.add<vector<edm::InputTag>>("recoEcalCandidateTags",{edm::InputTag("hltL1EG25Ele27WP85GsfTrackIsoFilter")});
> > +  desc.add<edm::FileInPath>("ESLookupTable",edm::FileInPath("EventFilter/ESDigiToRaw/data/ES_lookup_table.dat"));
> @rgerosa <https://github.com/rgerosa> Are the lookup tables for ES available in conditions via Global Tag?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/cmssw/pull/8112/files#r26058051>.
>